apps/api/jupyter.Dockerfile

```diff
@@ -27,6 +23,9 @@ RUN apt-get update && \
 ENV CPLUS_INCLUDE_PATH=/usr/include/gdal
 ENV C_INCLUDE_PATH=/usr/include/gdal
 
+ARG JUPYTER_REQUIREMENTS_FILE=jupyter-requirements.txt
+COPY $JUPYTER_REQUIREMENTS_FILE ./requirements.txt
+
```

Comment:
I move the COPY below so that it keeps apt cached when requirements.txt changes.